twingraph/src/main/kotlin/com/cosmotech/twingraph/api/TwingraphServiceImpl.kt

```diff
@@ -74,11 +71,13 @@ class TwingraphServiceImpl(
     if (findAllTwingraphs(organizationId).contains(graphId))
         throw CsmServerException("There is already a graph with the id : $graphId")
 
+    val version = "1"
```

Comment:
Could be interesting to make it a `const` named `INITIAL_VERSION` for further usage :thinking: 
Maybe the same thing can be done for `DEFAULT_GRAPH_ROTATION` set to 3